### PR TITLE
MS MARCO documents on ElasticSearch

### PR DESCRIPTION
**Pull request by HangCui0510**
Default BufferLimit:
Screen Shot 2020-05-04 at 11 17 35 AM

Increase BufferLimit:
Screen Shot 2020-05-04 at 11 37 26 PM

### CODE REVIEW

**Comment by HangCui0510** [comment]
@lintool, please review

File: src/main/java/io/anserini/search/SearchElastic.java [hunk 1]
```diff
@@ -73,6 +74,15 @@
   private final Args args;
   private RestHighLevelClient client;
 
+  private static final RequestOptions COMMON_OPTIONS;
+  static {
+      RequestOptions.Builder builder = RequestOptions.DEFAULT.toBuilder();
```
Comment (lintool) [inline src/main/java/io/anserini/search/SearchElastic.java:79]: should be two space indent here.

File: src/main/python/run_es_regression.py [hunk 2]
```diff
@@ -40,38 +41,51 @@ def is_alive(self):
...
+        else:
+            return True
 
     def does_index_exist(self, collection):
         # Make sure ES is alive:
         if self.is_alive():
             try:
-                response = requests.get('http://localhost:9200/{}'.format(collection))
+                response = requests.get(
+                    'http://localhost:9200/{}'.format(collection))
```
Comment (lintool) [inline src/main/python/run_es_regression.py:54]: I don't think this line is too long?

File: src/main/python/run_es_regression.py [hunk 3]
```diff
@@ -183,7 +227,8 @@ def evaluate(self, collection):
         else:
             logger.info('... appears to dead! :(')
     elif args.check_index_exists:
-        logger.info('Checking if index {} exists...'.format(args.check_index_exists))
+        logger.info('Checking if index {} exists...'.format(
```
Comment (lintool) [inline src/main/python/run_es_regression.py:230]: I don't think this line and the ones below are too long?

**Comment by lintool** [comment]
hey @eiston perhaps you could try and replicate this PR and confirm that it works?

Underlying page is here: https://github.com/castorini/anserini/blob/master/docs/elastirini.md

You should be able to do the MS MARCO passage replication as well.

**Comment by lintool** [comment]
@HangCui0510 broader point - avoiding changing lines that have nothing to do with the feature being implemented.. keeps the diffs clean and easier to understand.

**Comment by HangCui0510** [comment]
@lintool , My VS code changes the format automatically when I save, sorry I didn't notice it before creating this PR. I have created a new PR, please review.
@eiston , please refer to my new PR

**Comment by eiston** [comment]
> hey @eiston perhaps you could try and replicate this PR and confirm that it works?
> 
> Underlying page is here: https://github.com/castorini/anserini/blob/master/docs/elastirini.md
> 
> You should be able to do the MS MARCO passage replication as well.

ok

**Comment by lintool** [comment]
superseded by #1148

**Pull request closed without merging**